### PR TITLE
Add dev report for may 8th

### PR DESCRIPTION
**Pull request by vieux**

### CODE REVIEW

File: reports/2017-05-08.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Discourse is available at [forums.mobyproject.org](https://forums.mobyproject.org/) thanks to @thaJeztah. mailing-list mode is enabled, so once you register there, you will received every new threads / messages via email. So far, 3 categories were created: Architecture, Meta & Support. The last step missing is to setup an email address to be able to start a new thread via email.
+
+### Find a place for `/pkg`
+
+Lots of discussion and progress made on this [topic](https://github.com/moby/moby/issues/32989) thanks to @dnephin. [Here is the list](https://gist.github.com/dnephin/35dc10f6b6b7017f058a71908b301d38) proposed to split/reorganize the pkgs.
+
+### Find a good and non-confusing home for the remaining monolith
+
+@cpuguy is leading the effort [here](https://github.com/moby/moby/pull/33022). It's still WIP but the way we are experimenting with is to reorganise dcirectories within the moby/moby.
```
Comment (tiborvass) [inline reports/2017-05-08.md]: `cpuguy83`

Comment (cpuguy83) [inline reports/2017-05-08.md]: directories

Comment (vieux) [inline reports/2017-05-08.md]: done

File: reports/2017-05-08.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Development Report for May 08, 2017
+
+## Daily Meeting
+
+A daily meeting is hosted on [slack](dockercommunity.slack.com) every business day at 9am PST on the channel `#moby-project`.
```
Comment (AkihiroSuda) [inline reports/2017-05-08.md]: dockercommunity.slack.com -> https://dockercommunity.slack.com

File: reports/2017-05-08.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+During this meeting, we are talking about the [tasks](https://github.com/moby/moby/issues/32867) needed to be done for splitting moby and docker.
+
+## Topics discussed last week
+
+### The CLI split
+
+The Docker CLI was succesfully moved to [https://github.com/docker/cli](https://github.com/docker/cli) last week thanks to @tiborvass
+The dockercli is now compiled from the [Dockerfile](https://github.com/moby/moby/blob/a762ceace4e8c1c7ce4fb582789af9d8074be3e1/Dockerfile#L248)
```
Comment (AkihiroSuda) [inline reports/2017-05-08.md]: nit: dockercli -> Docker CLI or `docker/cli` ?

**Comment by tiborvass** [comment]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🦁

**Pull request merged**